src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier supplier = new ColorSupplier();
+    private final int bound = 101;
```

Comment:
what about the constant format https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
```suggestion
    private static final int BOUND = 101;
```